lldb/source/Target/SwiftLanguageRuntimeNames.cpp

```diff
@@ -179,7 +168,7 @@ static lldb::ThreadPlanSP GetStepThroughTrampolinePlan(Thread &thread,
   ConstString symbol_mangled_name = symbol->GetMangled().GetMangledName();
```

Comment:
I don't think you need symbol_mangled_name anymore.  The only place it gets used is to set symbol_name, and then we get a StringRef from it below.  But you could get that from symbol_name just as easily.